style.css

```diff
@@ -817,9 +833,38 @@ body {
     gap: 25px;
     position: absolute;
     right: 100px;
+    list-style: none;
   }
 
   .menu {
     display: none;
   }
+
+  .popup {
+    display: none;
+  }
+
+  #error {
+    display: none;
+  }
+
+  #card-work-1 {
+    display: block;
+    width: 100%;
+  }
```

Comment:
- [ ] Kindly make sure to meet the provided Figma as requested by the previous reviewer. :wink: 

   - ### Your designs
   ![image](https://user-images.githubusercontent.com/87197412/195839045-dd5bd824-9f48-45dc-8a98-c2d88a8dc802.png)

    ![image](https://user-images.githubusercontent.com/87197412/195839137-85300f76-1ace-4725-8579-9e0d401022de.png)


   - ### Figma designs
   ![image](https://user-images.githubusercontent.com/87197412/195839297-32d3968a-2b1f-486a-8618-1854ec775ee9.png)

   ![image](https://user-images.githubusercontent.com/87197412/195839338-3fe93abd-0a61-483e-a68c-325a6fe0e5c8.png)